apps/metastore/src/metastore/tests.py

```diff
@@ -79,22 +79,30 @@ def setUp(self):
   def test_show_tables(self):
     grant_access("test", "default", "metastore")
     with patch('beeswax.server.dbms.get') as get:
-      get.return_value = Mock(
-        get_databases=Mock(
-          return_value=['sfdc']
-        ),
-        get_database=Mock(
-          return_value={}
-        ),
-        get_tables_meta=Mock(
-          return_value=[{'name': 'customer'}, {'name': 'opportunities'}]
-        ),
-        server_name='hive'
-      )
-
-      response = self.client.post('/metastore/tables/sfdc?format=json')
-
-      get.assert_called()
+      with patch('beeswax.server.dbms.KazooClient') as KazooClient:
```

Comment:
Same we should keep the regular tests not using LLAP discovery as is (if they brake it probably means we are introducing a regression), and instead do new tests.

Reply:
Yes.. realized it late.  Made the changes and included 2 tests one in DBMS and another one in metastore.